docs/modules/migrate/pages/dmt-view-details.adoc

```diff
@@ -0,0 +1,30 @@
+= View Migration Details
+:description: Once the migration completes, you can view the details of the migration in a report, DMT log file, or the results IMap created on the target cluster.
+
+{description}
+
+The details of the migration are recorded as follows:
+
+* Migration report
++
+This is written to the _migration$$_$$report$$_$$[migration_id].txt_ file in the directory used when running the `dmt` command.
```

Comment:
I don't know these details by my heart. I think @yuce should review this PR as well to be sure that frontend related stuff. If nothing changed from the old content though, we should be fine. cc @rebekah-lawrence

Reply:
Invited Yuce to review; this will form part of the updates for v2 work (which I am starting now)